src/photos_model.py

```diff
@@ -301,7 +301,7 @@ def do_crop_activate(self):
         target_orientation = (un_orient_crop + self._orientation) % 360
 
         # crop overlay's rotation operation works at 90 degree intervals (clockwise)
-        num_rotations = target_orientation / 90
+        num_rotations = int(target_orientation / 90)
```

Comment:
Might be safer to add a "+ 0.5" prior to rounding from float to int.  Or, if this is really intended to be integer division, use `//` instead.